bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/GarbageCollectorThread.java

```diff
@@ -142,20 +157,21 @@ public GarbageCollectorThread(ServerConfiguration conf,
         throws IOException {
         this.gcExecutor = gcExecutor;
         this.conf = conf;
-
         this.entryLogger = ledgerStorage.getEntryLogger();
+        this.entryLogMetaMap = createEntryLogMetadataMap(conf.isGcPersistentEntrylogMetadataMapEnabled());
+
         this.ledgerStorage = ledgerStorage;
         this.gcWaitTime = conf.getGcWaitTime();
 
         this.numActiveEntryLogs = 0;
         this.totalEntryLogSize = 0L;
         this.garbageCollector = new ScanAndCompareGarbageCollector(ledgerManager, ledgerStorage, conf, statsLogger);
         this.gcStats = new GarbageCollectorStats(
-            statsLogger,
-            () -> numActiveEntryLogs,
-            () -> totalEntryLogSize,
-            () -> garbageCollector.getNumActiveLedgers()
-        );
+                statsLogger,
```

Comment:
it would be easier on reviewers if you are not making formatting changes on the existing code